### PR TITLE
Add `pipenv scripts` command to list scripts in Pipfile

### PR DESCRIPTION
**Pull request by GlenRSmith**
Enhancement discussed in #3686

### CODE REVIEW

**Comment by frostming** [comment]
A news fragment is needed to notice what is new. You can place it under the `news/` directory, see the other files about how to write one.

**Comment by uranusjr** [comment]
Maybe we should name this `scripts` instead for consistency. I misspelled `[scripts]` as `[script]` recently and spent almost half an hour wondering why my scripts don’t get picked up by Pipenv.

File: tests/integration/test_cli.py [hunk 1]
```diff
@@ -197,6 +197,19 @@ def test_bare_output(PipenvInstance):
...
+def test_scripts(PipenvInstance):
+    with PipenvInstance() as p:
+        with open(p.pipfile_path, "w") as f:
+            contents = """
+[scripts]
+pyver = "which python"
+            """.strip()
+            f.write(contents)
+        c = p.pipenv('scripts')
+        assert 'pyver' in c.out
```
Comment (uranusjr) [inline tests/integration/test_cli.py:210]: I would also test the `which python` part is also in the output.

**Review by frostming — CHANGES_REQUESTED** [review]
One more step before merge: add a news entry file under `news/` directory. @GlenRSmith

File: pipenv/cli/command.py [hunk 2]
```diff
@@ -716,5 +716,26 @@ def clean(ctx, state, dry_run=False, bare=False, user=False):
...
+    from ..core import project
+    if not project:
+        echo(u"project not found", err=True)
+        exit(1)
+    scripts = project.parsed_pipfile.get('scripts', {})
+    rpt = u"command\tscripts\n"
+    for k, v in scripts.items():
+        rpt += u"{0}\t{1}".format(k, v)
+    echo(rpt)
+    return
```
Comment (uranusjr) [inline pipenv/cli/command.py:737]: This return has no function.

Comment (GlenRSmith) [reply pipenv/cli/command.py]: Fixed.

**Comment by GlenRSmith** [comment]
@frostming Added. Sorry for the omission.

**Pull request merged**